### PR TITLE
Adds info about using adb to get error logs and other diagnostics

### LINKED ISSUE #56: Describe how to get bug reports off of Android
https://developer.android.com/studio/debug/bug-report.html is the ideal for Android 6+

There are also some notes at https://github.com/opendatakit/opendatakit/wiki/Collect-Troubleshooting

### PR DESCRIPTION
**Pull request by adammichaelwood**
closes #56 

addresses #402 

## What is included in this PR?

New section in collect-adb, with info on pulling debug logs.

Also, while I was there, I did some cleanup on the rest of the file.

### CODE REVIEW

File: src/collect-adb.rst [hunk 1]
```diff
@@ -1,36 +1,56 @@
...
+
+__ https://www.howtogeek.com/125769/how-to-install-and-use-abd-the-android-debug-bridge-utility_
+
+Standalone ADB
+~~~~~~~~~~~~~~~~
+
+You can also `install ADB without Android studio`__.
+This is not well supported, though,
+and should only be done
+if you cannot instal Android Studio on your computer.
```
Comment (yanokwa) [inline src/collect-adb.rst]: s/instal/install. Should the spell checker have caught this?

File: src/collect-adb.rst [hunk 2]
```diff
@@ -41,68 +61,80 @@ The forms are stored in :file:`sdcard/odk/forms/` folder on the device. They can
...
-Saving screenshot
-~~~~~~~~~~~~~~~~~~
+.. code-block:: console
+  
+  $  adb pull /sdcard/odk/database/{database-name}
+
+.. _saving-screenshot-with-adb:
 
-For taking a screenshot, run:
+Taking screenshota
```
Comment (yanokwa) [inline src/collect-adb.rst]: s/screenshota/screenshots

Comment (yanokwa) [reply src/collect-adb.rst]: Ugh. Ignore this. I think you fixed it in the next commit.

File: src/collect-adb.rst (hunk 1 above)
Comment (yanokwa) [reply src/collect-adb.rst]: Ugh. Ignore this. I think you fixed it in the next commit.

**Comment by adammichaelwood** [comment]
You seem to be reviewing one commit back. Those spelling errors were already fixed when this PR happened.

File: src/collect-adb.rst [hunk 3]
```diff
@@ -1,108 +1,205 @@
...
+in several SQLite databases, 
+which you can pull onto your local computer.
 
-Developers might also need to check the entries in the database from the computer. In such case pull the database file from the SD card and use any **SQLite visualizer** to view it. To pull the database into the computer, run:
+To see a list of available databases:
 
 .. code-block:: console
   
-  $  adb -pull /sdcard/odk/database/database.name
+  $  adb ls /sdcard/odk/database/
```
Comment (yanokwa) [inline src/collect-adb.rst]: There's no `/odk/database` folder. The databases live in `/odk/metadata`. I'm not sure it's always been like that but I'm pretty sure...

File: src/collect-adb.rst [hunk 4]
```diff
@@ -1,108 +1,205 @@
...
+You can also `install ADB without Android studio`__.
+This is not well supported, though,
+and should only be done
+if you cannot install Android Studio on your computer.
+
+__ https://android.gadgethacks.com/how-to/android-basics-install-adb-fastboot-mac-linux-windows-0164225/_
+
+.. _adb-forms:
+
+Managing Forms with ADB
```
Comment (yanokwa) [inline src/collect-adb.rst]: This should probably be lowercase forms.

File: src/collect-adb.rst [hunk 5]
```diff
@@ -1,108 +1,205 @@
...
+---------------------------
+
+Forms are stored on the device in  :file:`sdcard/odk/forms/`.
```
Comment (yanokwa) [inline src/collect-adb.rst]: Add a leading / on sdcard

File: src/collect-adb.rst [hunk 6]
```diff
@@ -1,108 +1,205 @@
...
+
+#. Type :kbd:`CTRL-C` to stop logging.
+
+You can then upload the :file:`logfile.txt` file to 
+a `Collect issue on GitHub <https://github.com/opendatakit/collect/issues>`_
+or post in the |forum|.
+
+.. _bugreport:
+
+Pull a bugreport
```
Comment (yanokwa) [inline src/collect-adb.rst]: I think bug report should be two words.

File: src/collect-adb.rst [hunk 7]
```diff
@@ -1,108 +1,205 @@
...
+To pull the video locally:
 
 .. code-block:: console
 
-  $ adb shell screenrecord /sdcard/example.mp4
+  $ adb pull /sdcard/video-name.png
+
+.. _adb-debug-logs:
+  
+Pulling logs for debugging
```
Comment (yanokwa) [inline src/collect-adb.rst]: Minor issue, but `pulling` seems like the wrong word. Capture feels better to me.

File: src/collect-adb.rst (hunk 3 above)
Comment (adammichaelwood) [reply src/collect-adb.rst]: I should have double checked on the device, but I got this from the original wiki troubleshooting guide...

**Pull request merged**